src/core/agent/services/connectionService.test.ts

```diff
@@ -310,13 +310,12 @@ describe("Connection service of agent", () => {
       message: "message",
     };
     const id = new Salter({}).qb64;
-    const now = new Date();
     await connectionService.createConnectionNote(connectionId, note);
     expect(updateContactMock).toBeCalledWith(connectionId, {
       [`note:${id}`]: JSON.stringify({
         ...note,
         id: `note:${id}`,
-        timestamp: now.toISOString(),
+        timestamp: new Date().toISOString(),
```

Comment:
This doesn't solve the flakey test I mentioned earlier. This date could now be slightly ahead of what happened in `createConnectionNote`.

Please just check that the timestamp exists, and stop checking the exact string value